client/src/components/kb/FeaturedList/FeaturedPageCard.tsx

```diff
@@ -38,11 +38,11 @@ const FeaturedPageCard = ({
           <Icon name="trash" size="small" />
         </div>
       )}
-      <div className="flex flex-col">
+      <div className="flex flex-col px-4">
         <p className="text-lg font-semibold flex-wrap">
           {truncateString(page.page.title, 50)}
```

Comment:
@hannahmcg Something else I just thought of - let's use "line-clamp-2" here instead of truncateString(). That way the text will always take up as much room as it can instead of being always cutoff at 50. https://tailwindcss.com/docs/line-clamp